faq/prooffacehelp.php

```diff
@@ -679,16 +679,42 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 </p>
 
 
+<h3>Search/Replace Popup</h3>
```

Comment:
We should consider adding an id to this `<h3>` and using it as an anchor in the help link.